examples/resources/admin.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# Example: Default Admin Role
+#
+kind: role
+metadata:
+  name: admin
+spec:
+  allow:
+    kubernetes_groups:
+    - '{{internal.kubernetes_groups}}'
+    logins:
+    - '{{internal.logins}}'
+    - root
+    node_labels:
+      '*': '*'
+    rules:
+    - resources:
+      - role
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - auth_connector
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - session
+      verbs:
+      - list
+      - read
+    - resources:
+      - trusted_cluster
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - event
+      verbs:
+      - list
+      - read
```

Comment:
Can we consider taking this giant list out? People tend to copy/paste it in and their admin roles get way more complicated to debug as a result.

Reply:
Good idea. I'm going to update it, and add everything into the role reference.